### ISSUE TITLE
Toggles for fast blocking of popups/scripts

### ISSUE THREAD

**Issue by ameshkov** [AdguardTeam/AdguardBrowserExtension#806]
https://github.com/AdguardTeam/AdguardBrowserExtension/issues/803#issuecomment-319152206

**Comment by Anoop2609** [comment]
remote font block

**Comment by The-Commissioner** [comment]
@Anoop2609 It takes a single rule to block remote fonts, don't think they would provide a feature for that:-

`||*$font,third-party`

**Comment by Anoop2609** [comment]
NO, It's not just about blocking remote fonts, i just give one option. You can use blocking remote fonts and 3rd party frames, images, media and many other. We should do "n"  numbers of things with Adguard. There is complete section of privacy which can be implemented #372 . I think we should add some little feature in any application because it gives user ability to customize. People like those application who give more feature. Combine all tiny feature you will get a effective product.

**Comment by snhv** [comment]
Not just elements (popup, script ...) but whole connection to that domain/subdomain too.
The way uBlock implement this is really the gold standard right now. It makes customized blocking for casual users a real piece of cake.
